src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -82,6 +82,9 @@ G'.adj_sub h
 /-- A subgraph is called a *spanning subgraph* if it contains all the vertices of `G`. --/
 def is_spanning (G' : subgraph G) : Prop := ∀ (v : V), v ∈ G'.verts
 
+lemma adj_sub' (G' : subgraph G) {u v : G'.verts} (h : ¬G.adj u v) : ¬G'.adj u v :=
```

Comment:
This is not a very explicit name. Could you find something better? `adj_sub_mt` would already be better IMO.

Also, that's outside the scope of this PR but using `sub` for `subgraph` seems *pretty* misleading.

Reply:
```suggestion
lemma adj_sub_mt (G' : subgraph G) {u v : G'.verts} (h : ¬G.adj u v) : ¬G'.adj u v :=
```

Yes, I agree it's kinda weird the name that was chosen. I didn't even realise that was what it was meant to mean until you said it.